extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -81,6 +83,8 @@ export const URL_EXPERIMENT_MAPPING = {
   '10': DOUBLECLICK_EXPERIMENT_FEATURE.REMOTE_HTML_EXPERIMENT,
   '11': DOUBLECLICK_EXPERIMENT_FEATURE.USDRUD_CONTROL,
   '12': DOUBLECLICK_EXPERIMENT_FEATURE.USDRUD_EXPERIMENT,
+  '13': DOUBLECLICK_EXPERIMENT_FEATURE.LAUNCH_CONTROL,
```

Comment:
LAUNCH is not exactly specific.  Can we make this DF_DEP_LAUNCH?

Reply:
done